src/helpText.jsx

```diff
@@ -1,12 +1,28 @@
 import React from 'react';
+import { isSafari } from './utils';
 
 /**
  * This file holds a few jsx constants to share helpText between different components.
  * Please keep it sorted for ease of use.
  */
 
+const publishDateHelp = (
+  <div>
+    {isSafari && <p>Format: Month(2 digits)/Day(2 digits)/Year(4 digits)</p>}
```

Comment:
I feel like all of these first messages should match across all of the date fields. Also here, you have it written as mm/dd/yyyy, but the placeholder is yyyy/mm/dd. That seems misleading or am I reading this wrong?

Reply:
missed it